### PR TITLE
WIP:  Allas documentation

### PR DESCRIPTION
**Pull request by galfthan**
To Do before merge (feel free to edit):

- [x]  Broken section links fixed
```
Link ./accessing_allas.md#protocols in file ./docs/data/Allas/terms_and_concepts.md on line 47 is broken
Link ../accessing_allas.md#protocols in file ./docs/data/Allas/using_allas/common_use_cases.md on line 18 is broken
Link ./using_allas/common_use_cases.md#using-allas-with-supercomputers in file ./docs/data/Allas/introduction.md on line 40 is broken
Link ./accessing_allas.md#accessing-allas-with-linux in file ./docs/data/Allas/introduction.md on line 46 is broken
Link ./accessing_allas.md#protocols in file ./docs/data/Allas/introduction.md on line 46 is broken
```

- [x]  Found broken file link Link ./swift,md in file ./docs/data/Allas/using_allas/s3_client.md on line 5 is broken

- [x]  Images are not shown

- [x]  Are we lacking some index files or something? By going to https://csc-guide-preview.rahtiapp.fi/origin/master_allas/#data/Allas the underlying folder structure is shown
I added an index file

- [ ]  Commands are ok and checked

- [x] Insert an entry point for the Allas material to the navbar (`mkdocs.yml`)
**We can probably insert a new top level category _data_ to the navbar**

- [x]  Nothing links to ./docs/data/Allas/using_allas/s3cmd.md
This file can be removed.

### CODE REVIEW

**Comment by Nortamo** [comment]
The following links are broken (404)
```
X Link ../../../cloud/pouta/install-client.md#configure-your-terminal-environment-for-openstack in file ./docs/data/Allas/using_allas/s3_client.md on line 74 is broken
X Link ../../../cloud/pouta/install-client.md#overview-of-openstack-command-line-tools in file ./docs/data/Allas/using_allas/swift_client.md on line 10 is broken
X Link ../../../cloud/pouta/install-client.md in file ./docs/data/Allas/using_allas/swift_client.md on line 12 is broken
X Link ../../../cloud/pouta/install-client.md#configure-your-terminal-environment-for-openstack in file ./docs/data/Allas/using_allas/swift_client.md on line 12 is broken
X Link ../../computing/disk-environment.md in file ./docs/data/Allas/introduction.md on line 40 is broken
X Link ../../computing/disk-environment.md in file ./docs/data/Allas/introduction.md on line 43 is broken
X Link /img/allas_projects_and_buckets.PNG in file ./docs/data/Allas/introduction.md on line 85 is broken
X Link /img/rahti_catalog_apache_spark.png in file ./docs/data/datanalytics/big-data-processing.md on line 15 is broken
X Link /accounts/creating-new-user-account/ in file ./docs/data/datanalytics/getting-started.md on line 7 is broken
X Link /accounts/joining-project/ in file ./docs/data/datanalytics/getting-started.md on line 7 is broken
X Link /accounts/adding-service-access-for-project/ in file ./docs/data/datanalytics/getting-started.md on line 7 is broken
X Link /connecting/ in file ./docs/data/datanalytics/getting-started.md on line 9 is broken
X Link /computing/module-system/ in file ./docs/data/datanalytics/getting-started.md on line 11 is broken
X Link /computing/module-system/#using-your-own-module-files in file ./docs/data/datanalytics/getting-started.md on line 11 is broken
X Link /img/datanalytics-data-in-process.png in file ./docs/data/datanalytics/getting-started.md on line 17 is broken
X Link /support/faq/how-to-move-data-between-puhti-and-allas/ in file ./docs/data/datanalytics/getting-started.md on line 34 is broken
X Link /support/faq/what-to-do-for-data-in-allas-after-project-ends/ in file ./docs/data/datanalytics/getting-started.md on line 50 is broken
X Link /computing/disk-environment/ in file ./docs/data/datanalytics/getting-started.md on line 66 is broken
X Link /data/#moving-data-between-csc-and-local-environment in file ./docs/data/datanalytics/getting-started.md on line 75 is broken
X Link /computing/batch-job-basics/ in file ./docs/data/datanalytics/getting-started.md on line 85 is broken
```

File: docs/data/Allas/allas_cpouta_change.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+We are releasing a new object storage service, **Allas**, which will be central for data management at CSC.
+
+You might wonder how this affects cPouta object storage users?
+
+In short, Allas is an evolution of cPouta object storage. If you are already using cPouta object storage, you can keep using it as you are currently doing. All your data will be carried over to Allas.
+
+<font color="red">
+CAVEAT: VERSIONED BUCKETS
+</font>
```
Comment (khappone) [inline docs/data/Allas/allas_cpouta_change.md]: This documetn needs to be updated

File: docs/data/Allas/introduction.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+Allas usage is founded on project based storage quotas. The default quota for a new project is 1 TB, but that can be increased if needed. Allas is the preferred storage site for any large datasets in CSC environment, so you should not hesitate to request larger quota for Allas, if you need to work with larger data sets. 
+
+ All the project members have equal access rights to the storage area that has been granted for the project. In practice, this means that if one project member uploads data to Allas, all the other project members can read, edit and also delete the data. Allas itself does not store any information about who has uploaded the data to Allas.
+
+The default quotas for every project are:
+
+| Resource | Limit |
+| :-------- |:------- |
+| Storage amount | 1 TiB |
```
Comment (khappone) [inline docs/data/Allas/introduction.md]: Quotas have not been updated. Correct quotas are here
https://wiki.eduuni.fi/display/CSCSR/Agenda+SR-JR+COMP+2019-09-05

File: docs/data/Allas/introduction.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+| Buckets per project | 1 000 |
+| Objects per bucket | 100 000 |
+
+
+Storing data in Allas consumes _billing units_. Accounting and billing information can be found under [Accounting principles and quotas](https://research.csc.fi/pouta-accounting){:target="_blank"}.
```
Comment (khappone) [inline docs/data/Allas/introduction.md]: This shoudl not link to pouta documentation any more

File: docs/data/datanalytics/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+[TOC]
+
+## Starting with data that is stored in Allas
+
+Just a quick reminder what Allas is.
+
+> **Allas** is part of the CSC storage portfolio and accessible from anywhere on the Internet. It is a storage service to host data for a project lifetime. The data from Allas can also be shared via Internet.
+> -- <cite>[Allas](/data/Allas/introduction/)</cite>
+
+Few things worth to notice about Allas is that it is meant to store and share data over the timeline of a project, maximum of three to five years. Default quota for a project is 1 TB but please keep in mind that used storage consumes project's billing units. Also bear in mind that no backups are taken of the data located in Allas, so please take care of backups of your data.
```
Comment (khappone) [inline docs/data/datanalytics/getting-started.md]: This also uses 1 TB for quota. Probably we should refer to the quota section for actual numbers so we don't get out-of-sync information

File: docs/data/Allas/introduction.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+In addition to these access flavors you can also see how to use Allas based on [Common Use Cases](./using_allas/common_use_cases.md){:target="_blank"}.
+
+
+&nbsp;
+
+
+## Billing and Quotas
+
+Allas usage is founded on project based storage quotas. The default quota for a new project is 1 TB, but that can be increased if needed. Allas is the preferred storage site for any large datasets in CSC environment, so you should not hesitate to request larger quota for Allas, if you need to work with larger data sets.
```
Comment (khappone) [inline docs/data/Allas/introduction.md]: Reference to 1 TB in text. Should probably be removed.

**Comment by Nortamo** [comment]
This pull request should be narrowed down to just the _Allas_ stuff  if we want to merge it into the master branch relatively quick. The moving data and datanalytics parts are not ready for merging.

**Comment by galfthan** [comment]
> This pull request should be narrowed down to just the _Allas_ stuff if we want to merge it into the master branch relatively quick. The moving data and datanalytics parts are not ready for merging.

This sounds like a plan. Can you @Nortamo  separate those into another branch and remove it from here

**Comment by Nortamo** [comment]
> > This pull request should be narrowed down to just the _Allas_ stuff if we want to merge it into the master branch relatively quick. The moving data and datanalytics parts are not ready for merging.
> 
> This sounds like a plan. Can you @Nortamo separate those into another branch and remove it from here

I'll do that.

**Comment by Nortamo** [comment]
Data analytics stuff moved to `data_analytics` branch
Data transfer stuff moved to `moving_data` branch

**Comment by Nortamo** [comment]
Commit 14ce5b633f9aa82d9f056795fb207d26d9245c9b broke some section links by removing the 
`-## Using Allas with Supercomputers` section
Can the links to this be removed or do they need to point to some other direction?

**Comment by galfthan** [comment]
Copied over the same sidebar structure we had in develop, now one can finally easily navigate the page in https://csc-guide-preview.rahtiapp.fi/origin/master_allas/

Some of the pages in "using allas" subfolder are now not in the sidebar, and it is a bit jarring when going there and loosing track of location in sidebar.  Perhaps add under clients

File: docs/data/Allas/using_allas/swift_client.md [hunk 6]
```diff
@@ -0,0 +1,380 @@
...
+```
+&nbsp;
+
+## Move objects
+
+You can copy data from one bucket to another with command ```swift copy```. The command below copies <i>file.txt</i> from _bucket1_ to _bucket2_.
+```bash
+swift copy --destination /bucket2 bucket1 file.txt
+```
+**Note:** In case there is no bucket called _bucket2_, Swift will create a new bucket with that name. However, even if there is a bucket called _bucket2_, Swift will claim that it created a new one even though it just copied the file to the already existing one:
```
Comment (galfthan) [inline docs/data/Allas/using_allas/swift_client.md]: There are a lot of these "manual" note things. We should probably use special note box syntax for most of these
!!!note

File: docs/data/Allas/using_allas/swift_client.md [hunk 7]
```diff
@@ -0,0 +1,380 @@
...
+Alternatively, you can set the project public and then access the file.
+
+
+&nbsp;
+
+
+
+## Files larger than 5 GB
+
+Since the OpenStack command does not support uploading files larger than 5GB, we can use Swift to upload a so-called _Static Large Object_ (SLO).
```
Comment (galfthan) [inline docs/data/Allas/using_allas/swift_client.md]: This is missleading. It does support uploading larger files and this section explains how to do it(!) Should be rephrased along the quote ne paragraph down

**Pull request merged**